src/types.ts

```diff
@@ -63,26 +63,22 @@ export interface ProtocolWithReturn<Data, Return> {
  * Extendable by user.
  */
 export interface ProtocolMap {
+  __crx_bridge_stream_open__: ProtocolWithReturn<{ channel: string; streamId: string }, boolean>
+  __crx_bridge_stream_transfer__: { streamId: string; action: 'transfer' | 'close'; streamTransfer: JsonValue }
```

Comment:
with some casting to internal signatures on `onMessage` and `sendMessage` we could remove those from here and hide them from the consumers

Reply:
I've been wanting to remove Stream from the `webext-bridge` core for sometime now. `Stream` can be extracted into a different package. I don't think people use this feature much, it increases the codebase footprint, which increases maintenance and documentation overhead.

Reply:
Yes, totally it'd be done as a separate PR, perhaps as part of v6.0.0 release. I was just throwing an idea out here. We'll remove it, release v6.0.0, and in the changelog, tell people that if they've been impacted by removal of `Stream`, create an issue. If we get enough issues, we'll release `Stream` as a new package.

Let me know your thoughts on this.